beacon-chain/blockchain/service.go

```diff
@@ -80,32 +78,23 @@ func (c *ChainService) Start() {
 	// to truly continue across sessions.
 	log.Info("Starting service")
 
-	genesis, err := c.beaconDB.GetCanonicalBlockForSlot(0)
+	var err error
+	c.genesisTime, err = c.beaconDB.GetGenesisTime()
```

Comment:
Is `GetGenesisTime()` necessary? Wouldn't it be easier to just use `params.GetConfig().GenesisTime`?

Reply:
This is important because in our implementation, if the client has no chainstate, genesis time will be time.Now(). Otherwise, the genesis time would be the one from the genesis block stored in the DB. We should always use the canonical genesis block's timestamp if it exists in persistent storage.

Reply:
To take a step back, shouldn't all the genesis blocks have the same timestamp? If node 1s genesis time is different than node 2s, they shouldn't be able to connect right?

Reply:
Ah, I didn't consider using `params.GetConfig().GenesisTime`, but I think Raul's right. This might be more of a bug in `params` though. Ideally, `params.GetConfig().GenesisTime` returns `time.Now()` if there is no genesis block, otherwise the timestamp in the genesis block.

Reply:
Forsure. The GenesisTime in config should be a future UNIX time of the genesis beacon chain block at slot 0. The use case is such all validators will be online before, and when it reaches `GenesisTime`,  the first slot gets proposed. 

I understand this is out of the scope of this PR, we can fix this later